diff --git a/dist/index.js b/dist/index.js
index c4327f7..1874743 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2006,7 +2006,8 @@ var TelegramAdapter = class {
         this.createAttachment("image", photo.file_id, {
           size: photo.file_size,
           width: photo.width,
-          height: photo.height
+          height: photo.height,
+          mimeType: "image/jpeg"
         })
       );
     }
